R/combinedscatterplot.R

```diff
@@ -523,9 +550,9 @@ processScatterSizes <- function(scatter.sizes, n) {
     sz.tmp
 }
 
-getOpacity <- function(opacity, scatter.sizes) {
+getOpacity <- function(opacity, scatter.sizes, fit.type) {
     if (is.null(opacity)) {
-        if (!is.null(scatter.sizes)) {
+        if (!is.null(scatter.sizes) || fit.type != "None") {
```

Comment:
I assume this is what we did before? Why is this needed?

Reply:
LabeledScatter didn't have this because it never allowed fit lines to be drawn. But Scatter has this code: https://github.com/Displayr/flipStandardCharts/blob/master/R/scatterchart.R#L417